content/md/products-super-power/enrich-your-reference-entity-records.md

```diff
@@ -0,0 +1,142 @@
+---
+id: enrich-your-reference-entity-records
+themes: products-super-power, manage-your-reference-entities
+title: "Enrich your reference entity records"
+popular: false
+ee-only: right
+related:
+---
+
+# Enrich your reference entity records
+
+## Add a record
+Once you’ve created a reference entity, you can add **records**.
```

Comment:
it is better to do not use contractions for verbs, this eases the reading.
"Once you have created"